### PR TITLE
Add "Google Chrome Profiles" extension

### PR DESCRIPTION
**Pull request by frouo**
## Description

List existing google chrome profile and launch the selected one.

## Type of change

New extension

## Screencast

https://user-images.githubusercontent.com/2499356/146422273-90f42940-7235-478c-bdb0-dd6190a36374.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @frouo 👋 

It looks awesome, I tested it out and only have a single comment 🔥 

It would be cool to have a HUD when opening a bookmark, It was possible to me to open it three times before I noticed it was due to lag it didn't open faster.

Could you look into that?

**Comment by frouo** [comment]
Oh yes, I got that too! Your suggestion will help thank you. I will do it shortly

**Comment by frouo** [comment]
Added new features:

new tab + search + custom filtering

preferences

File: extensions/google-chrome-profiles/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    View your Google Chrome profiles, show bookmarks, open the right profile.
+</p>
+
+## Intro
+
+With profiles, you can keep all your Chrome info separate, like bookmarks, history, passwords, and other settings.
+
+Problem: jumping from a profile to another is very painful.
+
+This is where this extensions comes in.
```
Comment (pernielsentikaer) [inline extensions/google-chrome-profiles/README.md:17]:
```suggestion
This is where this extensions come in.
```

File: extensions/google-chrome-profiles/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+| show profile's bookmarks   | `↵`   |
+| -------------------------- | ----- |
+| open profile chrome window | `⌘ ↵` |
+
+![show existing profiles](https://user-images.githubusercontent.com/2499356/146404653-37a398ea-338e-4124-89a8-f7583156f00c.jpg)
+
+## Show bookmarks
+
+| open and close raycast main window | `↵`   |
```
Comment (pernielsentikaer) [inline extensions/google-chrome-profiles/README.md:29]:
```suggestion
| open and close Raycast main window | `↵`   |
```

File: extensions/google-chrome-profiles/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+| ---------------------------------- | ----- |
+| open and keep raycast opened       | `⌘ ↵` |
```
Comment (pernielsentikaer) [inline extensions/google-chrome-profiles/README.md:31]:
```suggestion
| open and keep Raycast opened       | `⌘ ↵` |
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @frouo 👋 

Tried the new features out and they work as expected 💯 

I left you a few comments to look into, we're so close to merge 💎 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi @frouo 👋 

Thank you for the latest update, it works as expected 💪 

I'm going to accept it and then will my colleague do a final review before it's finally merged 💯

**Comment by mattisssa** [comment]
Hey! Let's get it merged 👏 
I have one question, what is the difference between simple "Open" and "Open in Background" here? For me it just always opens Google Chrome and makes it in focus.

CleanShot 2021-12-30 at 14 21 17@2x

**Comment by frouo** [comment]
@mattisssa 

> what is the difference between simple "Open" and "Open in Background" here?

"Open in google chrome" does launch google chrome + call `showHUD` (which should close the Raycast main window). 

"Open in Background" does launch google chrome + call `showToast` (which should keep open the Raycast main window if I am correct). That is not what you are experiencing?

**Pull request merged**